packages/backend/server/schema.prisma

```diff
@@ -422,6 +422,27 @@ model UserInvoice {
   @@map("user_invoices")
 }
 
+enum AiPromptRole {
+  system
+  assistant
+  user
+}
+
+model AiPrompt {
+  id        String       @id @default(uuid()) @db.VarChar
+  // prompt name
+  name      String       @unique @db.VarChar(20)
+  // if a group of prompts contains multiple sentences, idx specifies the order of each sentence
```

Comment:
hot to query multiple sentences prompts if their names are unique?

Reply:
alreadly fix this in next pr, let me cherry pick to this branch